concent_api/common/tests/test_api_view.py

```diff
@@ -16,13 +16,14 @@
 from golem_messages.factories.tasks import ComputeTaskDefFactory
 from golem_messages.utils import encode_hex
 
-from core.exceptions                import Http400
-from core.models                    import Client
-from common.constants                import ErrorCode
-from common.decorators               import require_golem_message
-from common.decorators               import handle_errors_and_responses
-from common.helpers                  import get_current_utc_timestamp
-from common.testing_helpers          import generate_ecc_key_pair
+from common.constants import ErrorCode
+from common.decorators import require_golem_message
+from common.decorators import handle_errors_and_responses
+from common.helpers import get_current_utc_timestamp
+from common.helpers import generate_uuid
+from common.testing_helpers import generate_ecc_key_pair
+from core.exceptions import Http400
+from core.models import Client
```

Comment:
This changes should be done in separate commit

Reply:
Agree, but in this situation I don't think it is necessary. For future- yes, let's do it in separate commits, for now- for me ok

Reply:
I think that if we want to force this kind of changes into separate commits people will do them less likely :) So maybe lets don't, but we may also discuss this with whole team if you want.